src/moclojer/core.clj

```diff
@@ -1,27 +1,32 @@
 (ns moclojer.core
   (:gen-class)
-  (:require [io.pedestal.http :as http]
+  (:require [clojure.edn :as edn]
+            [io.pedestal.http :as http]
             [io.pedestal.http.jetty]
-            [moclojer.handler :as handler]
             [moclojer.openapi :as openapi]
-            [yaml.core :as yaml]))
+            [yaml.core :as yaml]
+            [clojure.string :as str]
+            [moclojer.mrouter :as mrouter]))
+
+
+(defn parse []
```

Comment:
this function will always be used for route construction, since a package was created for routes `moclojer.mrouter`, we can move it there and call "only" the `mrouter/smart-router` in `moclojer.core`

leaving the core as clean as possible, the **moclojer** distribution will be a _binary (or jar)_ and we should leave only the server loading in the `moclojer.core`.

Reply:
good point.

Reply:
move it